website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -41,12 +41,16 @@
         <br />
         <br />
         <div class="row" data-bind="if: showSelect">
-          <div class="col-md-8">
+          <div class="col-md-6">
             <select class="form-control" id="s3_bucket" name="s3_bucket"
                     data-bind="value: selectedBucket,
                                attr.disabled: !loadedBucketList(),
                                options: bucketList"> </select>
           </div>
+          <div class="col-md-3">
```

Comment:
Markup seems off here, too. I think you need a form-group and "Encrypt file uploads" should be a proper `label`. I wonder if this will fix the alignment issue here:

![osf___s3_update_settings](https://cloud.githubusercontent.com/assets/2379650/8993065/933e6508-36d0-11e5-9b15-03d53cca85e2.png)